### PR TITLE
added BanPlayer and deprecated ExploitBan

### PR DESCRIPTION
**Pull request by r0adra93**
## Description

added BanPlayer to the exports.lua

deprecated out ExploitBan as it is unnecessary.

deprecated out ExploitBan by sending it to the BanPlayer function to keep it functional for scripts that use ExploitBan.

## Checklist

- [x] I have personally loaded this code into an updated qbcore project and checked all of its functionality.
- [x] My code fits the style guidelines.
- [x] My PR fits the contribution guidelines.

### CODE REVIEW

File: server/exports.lua [hunk 1]
```diff
@@ -317,19 +317,25 @@ end
...
-    TriggerEvent("qb-log:server:CreateLog", "anticheat", "Anti-Cheat", "red", name .. " has been banned for exploiting " .. origin, true)
+    TriggerEvent('qb-log:server:CreateLog', resource, 'Player Banned', 'red', string.format('%s was banned by %s for %s', name, resource, reason), true)
+    DropPlayer(src, string.format("You were permanently banned by the server for: %s",reason))
 end
+exports("BanPlayer",BanPlayer)
 
-exports('ExploitBan', ExploitBan)
+-- Deprecated exploit ban
+local function ExploitBan(playerId, origin)
+    BanPlayer(playerId,origin,"2147483647",'Anti Cheat')
```
Comment (charj) [inline server/exports.lua:339]: What's the number as a string?

Comment (charj) [reply server/exports.lua:339]: I'm guessing it's an epoch to ban a user infefinately..
Just looks like a weird number is all

I realise you've only moved it here btw, just interested.

Comment (r0adra93) [reply server/exports.lua:339]: That's exactly what it is. I could take it out of quotes.

Comment (Z3rio) [reply server/exports.lua:339]: After some googling, it also seems like its the largest possible int32 number.

Comment (r0adra93) [reply server/exports.lua:339]: It is the epoch date to release ban...

**Pull request closed without merging**